packages/timestamp/src/Timestamp.js

```diff
@@ -46,6 +46,9 @@ export default class Timestamp extends Component {
       // # 1 year
       distance = Math.round(timeDifference / (60 * 60 * 24 * (365 / 12)));
       timePassed = `${distance} ${pluralize("month", distance)}`;
+    } else {
```

Comment:
I think we should follow the rest of this block and use an `else if` instead of just `else`, as that way we won't have anything unaccounted for end up using the `else`.

Reply:
@dylankenneally please rebase your commits and follow our commit guidelines: https://github.com/Autodesk/hig/blob/development/DEVELOPING.md#git-commits

Once this has come back with the requested changes I can get the visual-tests running.

Reply:
Thanks for getting back to me @wmui51. I'm going to need more information though.

> please rebase your commits
- Is this what you mean?
![image](https://user-images.githubusercontent.com/12589363/66439508-9e58af80-ea7b-11e9-93de-6e486ae1e693.png)

> follow our commit guidelines
- As in the [Angular commit messages format](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commit-message-format)? Or is there something else you need from me? 

> I think we should follow the rest of this block and use an `else if` instead of just `else`, as that way we won't have anything unaccounted for end up using the `else`.

My use of `else` was deliberate.
1. what would be the `if` condition? Unless we're thinking that we would also offer things like "decades" and "centuries", which I don't think we need, I can't see what the `if` would be?
1. unless there was a catch-all `else` (as I've put in the code), it would be conceivable for `timePassed` to be undefined - thats the very bug I'm fixing here.

Reply:
Your `development` branch is out of date, so usually I would do an updated pull of that and then do a `git rebase` to update my local branch. While doing that, you can squash all your commits into one since they are all pertaining to the same thing.

Yes, I meant the Angular commit messages format, for your commit it would probably be a `fix:` commit, without the proper formatting the changes will not be picked up by semantic release.

For the `if else` part I was thinking something like `timeDifference > 60 * 60 * 24 * 30 * 12` might work. I feel like having `timePassed` be `undefined` might not be a bad thing as that can allow us to know of another case that needs to be added/fixed. But if the above `if` doesn't make sense, we can go ahead with you catch-all `else`.

Reply:
I've pulled from Autodesk `development` to my branch and pushed it up to pick up yesterdays merged in code to `development`.

I'm happy with my commit messages as they are on my branch, but I do understand the teams desire to have a consistent set in the development branch. How about when/if you merge this in, you use squash if that's desired?
![image](https://user-images.githubusercontent.com/12589363/66526024-07efc100-eb43-11e9-886e-c141f7976427.png)

And you can set the squashed commit message as you see fit?
![image](https://user-images.githubusercontent.com/12589363/66526064-2bb30700-eb43-11e9-9a12-4b01823012a8.png)

Reply:
However, before I could squash the commits I need the first commit to follow the Angular commit guidelines.